app/Http/Requests/ColumnMonsterRequest.php

```diff
@@ -0,0 +1,55 @@
+<?php
```

Comment:
I think we can remove this `ColumnMonsterRequest.php` file entirely - it's not used anywhere, right?